pkg/cli/root.go

```diff
@@ -42,6 +42,20 @@ var rootCmd = func() cli.Command {
 					},
 				}
 			},
+			"cli": func() cli.Command {
+				return &cli.RootCommand{
+					Name:        "cli",
+					Description: "Perform operations related to the CLI request",
+					Commands: map[string]cli.CommandFactory{
+						"do": func() cli.Command {
+							return &CLIHandleCommand{}
+						},
+						"cleanup": func() cli.Command {
+							return &CLIHandleCommand{Cleanup: true}
```

Comment:
Should do and cleanup its its own command instead of using a bool to differentiate between do and cleanup? I understand do and cleanup share a lot of common logic, maybe we can refactor those logic as a util?

Reply:
I wanted to follow the pattern of IAMRequest. Handle command handles IAM/CLI request. the difference is CLI handle has two parts, do and cleanup. We can do a wrapper layer, ```"do": &CLIDoCommand{CLIHandleCommand{dofunc(ctx, *v1alpha1.CLIRequest) error}}```,  ```"cleanup": &CLICleanupCommand{CLIHandleCommand{cleanupfunc(ctx, *v1alpha1.CLIRequest) error}}```, but it is kinda redundant.

Reply:
Is it possible that the CLI handle command will need to support more commands besides cleanup and do in the near future? If in the near future, we will only have do and cleanup, I'm fine with current structure.

Reply:
Yes, based on the design cli requests only have do and cleanup, I also cannot think of any possible additional features besides these two. If needed, we can come back and refactor.